src/v2/api/controllers/UserController.ts

```diff
@@ -56,6 +56,7 @@ import { UpdateSuperheroDTO } from '../dtos/UpdateSuperheroDTO';
 import { DisciplineIdsResponse } from '../responses/DisciplineResponse';
 import { SuperheroResponse } from '../responses/SuperheroResponse';
 import { ApiEndpoint } from 'src/v2/utils/documentation/decorators';
+
```

Comment:
I can't add it to 148 of code, so I will leave it here. You should move TelegramGuard from ApiEndpoint to UseGuard, because currently TelegramGuard doesn't work with ApiEndpoint. And you should do it to all TelegramGuard in this file

Reply:
Not my endpoint